src/utils/PageEmbed.ts

```diff
@@ -183,10 +202,14 @@ export class PageEmbed extends EmitterBase
                 return;
             }
 
-            this.emit("press", int, type);
```

Comment:
I believe this isn't really needed since the control buttons are handled by the class itself and the user doesn't need to know when they're pressed. But let me know if i'm wrong

Reply:
My idea was to have it like this so you can implement custom behavior, for example if you don't need a message content, which is required in the constructor.